client/blocks/get-apps/apps-config.tsx

```diff
@@ -160,17 +160,17 @@ export const createWordPressStudioConfig = (
 			[ PlatformType.MacIntel ]: {
 				...platformConfigs[ PlatformType.MacIntel ],
 				onClick: () => recordTracksEvent( 'calypso_studio_download_mac_click' ),
-				link: 'https://cdn.a8c-ci.services/studio/studio-darwin-x64-v1.3.0.dmg',
+				link: 'https://cdn.a8c-ci.services/studio/studio-darwin-x64-v1.3.2.dmg',
```

Comment:
This is not in relation to this specific PR, but a general idea:

I wonder if it would make sense to create three redirect links (each for its platform (Mac Intel, Mac Apple Silicon, Windows)) and then just change the redirect target with each Studio release.

With that we could avoid having to update the links everywhere (Studio site download page English and Spanish version, /me/get-apps page) every two weeks.

Or, maybe there's a way to change some settings when the files are created in Buildkite.

Reply:
I created https://github.com/Automattic/dotcom-forge/issues/10306 to track this idea 👍